src/lib.rs

```diff
@@ -29,31 +31,44 @@ struct CategoryOriginal {
 #[serde(rename_all = "camelCase")]
 pub struct DeserializablePubkey(#[serde(with = "field_as_string")] pub Pubkey);
 
+fn default_amplification_ratio() -> Decimal {
+    dec!(1.5)
+}
+
 #[derive(Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct DefaultSlippage {
+
+pub struct DefaultAutoSlippage {
```

Comment:
what is this? why `Auto` naming

Reply:
bad naming i guess, renamed to `Default`

Reply:
Sorry, i changed it again because there was the same mistake in jupiter-core-rs but `Default` cannot work over there, so it is now `DefaultDynamicSlippage`